api/db_connect.py

```diff
@@ -0,0 +1,31 @@
+"""
+    module to initialize db connections 
+    and close them
+"""
+import psycopg2
+from .secret import DB_USER, DB_PASSWORD
+from instance.config import app_config
+
+class DBConnect():
+    def __init__(self):
+        """initialize db instance"""
+        self.conn = ""
+        self.cursor = ""
```

Comment:
In python, there is no need for you to declare variables beforehand as you've done in line 11 and line 12